kratos/containers/distributed_csr_matrix.h

```diff
@@ -247,60 +328,114 @@ class DistributedCsrMatrix
 
     IndexType size2() const
     {
-        return GetColNumbering().Size(); 
+        return GetColNumbering().Size();
     }
 
-    inline IndexType local_nnz() const{
+    inline IndexType local_nnz() const
+    {
         return mDiagonalBlock.nnz();
     }
 
-    const DataCommunicator& GetComm() const{
+    const DataCommunicator& GetComm() const
+    {
         return mrComm;
     }
 
-    inline CsrMatrix<TDataType,IndexType>& GetDiagonalBlock(){
+    inline CsrMatrix<TDataType,IndexType>& GetDiagonalBlock()
+    {
         return mDiagonalBlock;
     }
-    inline CsrMatrix<TDataType,IndexType>& GetOffDiagonalBlock(){
+    inline CsrMatrix<TDataType,IndexType>& GetOffDiagonalBlock()
+    {
         return mOffDiagonalBlock;
     }
 
-    inline const CsrMatrix<TDataType,IndexType>& GetDiagonalBlock() const{
+    inline const CsrMatrix<TDataType,IndexType>& GetDiagonalBlock() const
+    {
         return mDiagonalBlock;
     }
-    inline const CsrMatrix<TDataType,IndexType>& GetOffDiagonalBlock() const{
+    inline const CsrMatrix<TDataType,IndexType>& GetOffDiagonalBlock() const
+    {
         return mOffDiagonalBlock;
     }
 
+    const std::map<IndexType, IndexType>& GetOffDiagonalLocalIds() const
+    {
+        return mOffDiagonalLocalIds; //usage: mOffDiagonalLocalIds[global_id] contains the local_id associated to that global_id (for a off diagonal block entry)
+    }
 
+    std::map<IndexType, IndexType>& GetOffDiagonalLocalIds()
+    {
+        return mOffDiagonalLocalIds; //usage: mOffDiagonalLocalIds[global_id] contains the local_id associated to that global_id (for a off diagonal block entry)
```

Comment:
those comment should go to the doc of the function

Reply:
i disagree ... it is good that it is also here